lib/variable.js

```diff
@@ -12,15 +12,28 @@ const base64Regex = /^([A-Za-z0-9+/]{4})*([A-Za-z0-9+/]{4}|[A-Za-z0-9+/]{3}=|[A-
  * @param  {Object} extraAccessors Extra accessors to install.
  * @return {Object}
  */
-module.exports = function getVariableAccessors (container, varName, defValue, extraAccessors) {
+module.exports = function getVariableAccessors (container, varName, extraAccessors) {
   let isBase64 = false
+  let isRequired = false
+  let defValue
+  let example
 
   /**
    * Throw an error with a consistent type/format.
    * @param {String} value
    */
   function raiseError (value, msg) {
-    throw new EnvVarError(`"${varName}" ${msg}, but was "${value}"`)
+    let errMsg = `"${varName}" ${msg}`
+
+    if (value) {
+      errMsg = `${errMsg}, but is set to "${value}"`
+    }
+
+    if (example) {
+      errMsg = `${errMsg}. An example of a valid value would be "${example}"`
```

Comment:
Maybe not a big deal, but wrapping the example in quotes might look weird weird without escaping the quotes in the example string, particularly for json objects. :shrug

Reply:
I wasn't sure about this one. Perhaps change it to something like `An example of a valid value would be: ${example}`, i.e remove quotes, but add a colon

Reply:
Maybe this?
```suggestion
      errMsg = `${errMsg}. An example of a valid value would be ${JSON.stringify(example)}`
```